conformance/README.md

```diff
@@ -608,33 +610,63 @@ expectation, ending that branch of the test tree.
-                |  "("  "clob"    bytes*        ")"
-
-// TODO Other types per denotational semantics
-
-codepoint     ::=  int                               // in the range 0..0x10FFFF
-
-model-symtok  ::=  string
-                |  int
-                |  "("  "text"  codepoint*  ")"
-                |  "("  "absent"  string  int  ")"       // symtab name + offset
-
-model-field   ::=  "("  model-symtok  model-value  ")"
-
-annotated     ::=  "("  "annot"  model-content  string*  ")"
+model-value     ::=  model-content  |  annotated
+
+model-content   ::=  null.null
+                  |  bool
+                  |  int
+                  |  string
+                  |  "("  "null"      model-type       ")"
+                  |  "("  "bool"      bool             ")"
+                  |  "("  "int"       int              ")"
+                  |  "("  "float"     string           ")" // See https://amazon-ion.github.io/ion-docs/docs/float.html
+                  |  "("  "decimal"   model-decimal    ")"
+                  |  "("  "timestamp" model-timestamp  ")"
+                  |  "("  "string"    codepoint*       ")"
+                  |  "("  "symbol"    model-symtok     ")"
+                  |  "("  "list"      model-value*     ")"
+                  |  "("  "sexp"      model-value*     ")"
+                  |  "("  "struct"    model-field*     ")"
+                  |  "("  "blob"      bytes*           ")"
+                  |  "("  "clob"      bytes*           ")"
+
+codepoint       ::=  int                                   // in the range 0..0x10FFFF
+
+model-symtok    ::=  string
+                  |  int
+                  |  "("  "text"  codepoint*  ")"
+                  |  "("  "absent"  string  int  ")"       // symtab name + offset
+
+model-field     ::=  "("  model-symtok  model-value  ")"
+
+model-decimal   ::= int int                                // coefficient + exponent
+                  | "(" "negative_0" ")" int
+
+// All timestamp subfields are interpreted as UTC time.
+// I.e. the following timestamps are not equivalent, but they represent the same point in time.
+// (timestamp (precision second) 1 2 3 (offset -1440) 4 5 6)
+// (timestamp (precision second) 1 2 3 (offset +1440) 4 5 6)
+
+model-timestamp ::= prec-year     int
+                  | prec-month    int int
+                  | prec-day      int int int
+                  | prec-minute   int int int offset int int
+                  | prec-second   int int int offset int int int
+                  | prec-fraction int int int offset int int int model-decimal
+
+prec-year       ::= "("  "precision"  "year"   ")"
+prec-month      ::= "("  "precision"  "month"  ")"
+prec-day        ::= "("  "precision"  "day"    ")"
+prec-minute     ::= "("  "precision"  "minute" ")"
+prec-second     ::= "("  "precision"  "second" ")"
+prec-fraction   ::= "("  "precision"  int      ")"         // positive, count of digits of fractional part of seconds
```

Comment:
Isn't the count of fractional digits implicit in the model-decimal?

Reply:
Yes, but also in general, all precision is implicit given the number of fields that are present. Based on your prior feedback about making the precision explicit, I figured you would want to have this explicit as well, and I didn't particularly care one way or the other.